<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 84, 84);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="create_btn">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 44, 192);
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: rgb(180, 59, 148);
min-width: 10em;
padding: 4px;</string>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="edit_btn">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 44, 192);
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: rgb(180, 59, 148);
min-width: 10em;
padding: 4px;</string>
        </property>
        <property name="text">
         <string>Изменить</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 44, 192);
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: rgb(180, 59, 148);
min-width: 2em;
min-height: 2em;
QTableCornerButton::section {
    background-color: rgb(255, 44, 192);
	border-style: outset;
	border-width: 2px;
	border-radius: 10px;
	border-color: rgb(180, 59, 148);
	min-width: 2em;
	min-height: 2em;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
